src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -632,7 +638,7 @@ public boolean getShowGitToolOptions() {
         }
 
         public List<SCMSourceTrait> getTraitsDefaults() {
-            return Collections.singletonList(new BitbucketBranchDiscoveryTrait());
+            return Arrays.asList(new BitbucketBranchDiscoveryTrait());
```

Comment:
why this change? (not a problem just trying to understand)

Reply:
old accidental holdover from when i added tags as a default trait for some testing. Will change back